Misc/NEWS.d/next/Tests/2021-09-14-13-16-18.bpo-45195.EyQR1G.rst

```diff
@@ -0,0 +1,3 @@
+Fix test_readline.test_nonascii(): sometimes, the newline character is not
```

Comment:
I am just confused about this New entry. The end user should notice this New entry?